### DISCUSSION TITLE
How am I supposed to run macOS worker node?

### DISCUSSION THREAD

**Discussion by rkxx08** [concourse/concourse#6638] [Help & Support]
The manual says to download and run `concourse` executable, but I can't run it on my macOS instance as the executable is not code-signed. (security policy reject it) 

Am I supposed to run Concourse on macOS via Docker? Or running the executable directly?

**Comment by taylorsilva** [comment]
Running via docker will probably be easier for you. As a Concourse dev on macOS, I'm always running Concourse within docker. I imagine you may run into the same problem with `fly`, which is something I build locally of course so I don't get the code signing issue.
